tests/src/test/java/alluxio/hadoop/HdfsFileInputStreamIntegrationTest.java

```diff
@@ -410,7 +410,7 @@ public void positionedReadCache() throws IOException, AlluxioException {
   @Test
   public void positionedReadCacheNoPartialCache() throws IOException, AlluxioException {
     createUfsInStreamNoPartialcache(ReadType.CACHE);
-    mUfsInputStream.readFully(0, new byte[FILE_LEN]);
+    mUfsInputStream.readFully(0, new byte[FILE_LEN - 1]);
```

Comment:
This should probably be min of block length - 1 and file length - 1, but I don't think there is a concept of block in this class yet.